rdr_service/resource/generators/participant.py

```diff
@@ -595,6 +692,7 @@ def _prep_modules(self, p_id, ro_session):
 
                         module_data['status'] = module_status.name
                         module_data['status_id'] = module_status.value
+                        mod_ca['ConsentAnswer'] = consent['consent_value']
```

Comment:
This question answer value goes into to the activity log so you can know what they answered for each consent module.

Reply:
@robabram I don't have a problem with this, but since we have the status  in the module data as well (SUBMITTED -- aka yes if this is a consent module, SUBMITTED_NO_CONSENT, SUBMITTED_NOT_SURE) just wondering if we also need the answer code or can work off the module status and response status (IN_PROGRESS or COMPLETED).  Will approve as is for now.  Given that RDR tends to use those SUBMITTED status enums in `participant_summary`.